js/common.js

```diff
@@ -0,0 +1,8 @@
+const $ = (select, el = document) => el.querySelector(select);
+
+const SIGN_INFO_REGEX = {
+  email: /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/,
+  password: /^[A-Za-z0-9]{8,15}$/,
+};
```

Comment:
요렇게 객체의 변경을 막아주는게 좋을 것 같아요!

```suggestion
const SIGN_INFO_REGEX = Object.freeze({
  email: /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/,
  password: /^[A-Za-z0-9]{8,15}$/,
});
```